book/website/reproducible-research/licensing/licensing-ml.md

```diff
@@ -1,10 +1,31 @@
 (rr-licensing-ml)=
-# Machine Learning Model Licenses
+# Licensing Machine Learning models
+
+## Legal Status of AI/ML model weights
+
+It is an open question wether AI/ML models weights are even copyrightable.
+US copyright law specifically excludes the following from works eligible for copyright protection: "any idea, procedure, process, system, method of operation, concept, principle, or discovery, regardless of the form in which it is described, explained, illustrated, or embodied in such work."
+In addition the US Copyright Office has stated that this exclusion extends to "scientific or technical methods or discoveries;" "mathematical principles;" and "formulas or algorithms."
+
+A legal doctrine known as the [idea–expression distinction/dichotomy](https://en.wikipedia.org/wiki/Idea%E2%80%93expression_distinction) draws a distinction between an abstract idea and a specific implementation, where code authored by a human that implements an algorithm is copyrightable as a creative work of authorship but the algorithm itself is not.
+
+It is also generally accepted in US copyright law that to be copyrightable a work must be a product of human creative authorship and not that of an automated process.
+
+It is not obvious if model weights would be considered works of human authorship or rather the results of automated processes, as a general principle or discovery or as a specific implementation.
+These questions will need to be decided on by the courts and/or legislated on before the status of model weights can be properly established.
+
+In the absence of clarity on this point many organisations are taking the calculated risk of operating under the assumption that they are copyrightable and generally treating them similarly to datasets and/or software.
+Some companies are offering legal protections for users of services based on these systems, offering to cover legal expenses arising from challenges to the copyright status of the outputs of these systems.
+It seems likely that some form of copyright-like protection will be extended to model weights but it's precise contours are yet to be determined.
```

Comment:
Why does this seem likely? I got the impression that it is going the other way around?

Reply:
Perhaps likely is overstating it, it is a matter of opinion and under current law it does actually seem somewhat unlikely. I emphasize as a non-lawyer, the precedents mostly seem to lean against it but this particular question does not seem to have been tested in court. It seems to me to be a bit of a 'luck of the draw' when it comes to judges how much they will consider the previous decisions analogous to a new one depending on their level of understanding of these systems.

The IPO and various other parties, well summarized in the unflattering moniker the 'copyright cartel', have considerable stakes in being able to monetize AI/ML models It may be sufficient for them to do so relying on trade secrets but they may want stronger legal guarantees of protection. 'Likely' is born of a certain pessimism that the trend towards more and more expansive IP protections which serve to entrench existing monopolies is going to be reversed anytime soon.

See: https://ipo.org/wp-content/uploads/2020/11/SG-model-rights-committee-paper-pub.pdf

Given the dysfunction of so many of our legislative bodies, however, they may struggle to pass comprehensive new protections for these systems. They may also be averse to opening up the question of IP law reform in influential jurisdictions like the US or EU lest doing so backfire and reduce the protections that they now enjoy.

Reply:
Rephrased to "It is possible that some form of copyright-like protection will be explicitly extended to model weights but it's precise contours are yet to be determined." in [28aa8a7](https://github.com/the-turing-way/the-turing-way/pull/3142/commits/28aa8a7af420820fed4e4cbf7c9152e6e7c836c4)